src/Collector/FormCollector.php

```diff
@@ -0,0 +1,156 @@
+<?php
```

Comment:
How much of this code is duplicated with componentFinder? Maybe create Abstract parent?

Reply:
I think it's just finding of createComponent method